variables.tf

```diff
@@ -3,10 +3,22 @@ variable "cf_org_name" {
   type        = string
 }
 
+variable "cf_space_name" {
+  description = "Cloudfoundry SPACE name to use for deploying all Thanos components. If empty, the module will create it's own space"
+  type        = string
+  default     = ""
+}
+
+variable "cf_region" {
+  description = "The HSP region to which this will be deployed"
+  type        = string
+  default     = "us-east"
+}
+
 variable "cf_app_domain" {
```

Comment:
If this is deprecated how do we set the domain to use for routes?

If it is deprecated we should just remove it and then the consumer can take action on the error.

Reply:
Domain is derived from hsdp provider region settings. As you suggested, variable will be removed completely.